src/labeler.ts

```diff
@@ -40,6 +40,23 @@ export async function run() {
 
     const labels: string[] = [];
     const labelsToRemove: string[] = [];
+
```

Comment:
Could you please move this right after the following line?
`const changedFiles: string[] = await getChangedFiles(client, prNumber);`